docs/usage.md

```diff
@@ -3,130 +3,441 @@
+- **Malicious**: If the reputation returned by VirusTotal is *Malicious*.
+- **Suspicious**: If the reputation returned by VirusTotal is *Suspicious*.
+- **Good**: If the reputation returned by VirusTotal is anything but *Suspicious*.
 
-Use FortiAI to help with this task by clicking the **Response Plans / Jinja / Playbook How To's** option in the FortiAI flyout.
+### Creating a Playbook Using Playbook Designer
 
-Typing your requirements, such as
+1. **Initiate Playbook Designer**: To learn more about working with Playbook Designer, refer to the section [Creating Playbooks](https://docs.fortinet.com/document/fortisoar/7.6.0/playbooks-guide/331279/introduction-to-playbooks#Creating_Playbooks) in the Playbooks Guide of FortiSOAR product documentation.
 
->*I need help with Jinja to trim HTML from a string input*
+   - From the **Playbook Designer**, type your requirements in a conversational format. For example:
 
-Click the **Post** icon. FortiAI, in response, provides you with the relevant Jinja to utilize:
+    >*Extract Indicators from the Alert Source Data and enrich them using VirusTotal. Based upon reputation, if `malicious` update the indicator reputation to `malicious` else if `suspicious` update the indicator reputation to `suspicious` else update indicator reputation to good.*
 
-![FortiAI response for a Jinja query](../docs/res/FortiAIJinja.png)
+2. **Modify Playbook Outline**:
+   - Adjust the playbook outline as needed, such as changing the integration tool from VirusTotal to IPStack.      
+
+    ![FortiAI - Suggested playbook outline](../docs/res/FortiAIPBGeneration.png)
+
+3. **Generate Playbook**:
+   - Once satisfied with the outline, click **Ok, Generate Playbook**.
+
+>[!NOTE]
+>If a step uses a connector that is not installed, FortiAI informs you with a message indicating that the specific step requires a connector that is not installed on the system.
+
+4. **Monitor Playbook Generation**:
+   - A loader image indicates the progress while FortiAI is generating playbook steps.
+   
+5. **Review Generated Playbook**:
+   - After the steps have been generated, a suggested playbook block appears in the *Playbook Designer*. You can review each step by double-clicking it and examining the details.
+
+    ![Reviewing a playbook step generated by FortiAI](../docs/res/FortiAIGeneratedPb_StepReview.png)
+
+6. **Integrate and Finalize Playbook**:
+   - Connect the reviewed AI-generated playbook blocks to the main playbook flow to create a complete workflow.
+
+The ability to generate playbooks through OpenAI's assistant represents a significant advancement in playbook design, offering immense assistance to those new to developing playbooks. By following the above steps, administrators can efficiently create and manage robust playbooks to address various threat mitigation scenarios.
+
```

Comment:
Added below tip;

>[!TIP]
>It is recommended to clear a conversation whenever user is intended to execute different usecases. It will also help in save tokens being consumed.

Reply:
Done